msteams-platform/concepts/deploy-and-publish/appsource/submit-your-app.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Submit your app to Partner Center
+description: Learn how to create a Partner Center account and submit your app for store validation.
+author: heath-hamilton
+ms.author: lajanuar
+ms.topic: how-to
+---
+# Submit your Microsoft Teams app to Partner Center
+
+After preparing your store submission, you can officially submit your app for review.
+
+## 1. Create a Partner Center account
+
+To publish your app to the Teams store and AppSource, you must [first set up a developer account](https://docs.microsoft.com/office/dev/store/open-a-developer-account).
+
+## 2. Submit your app
+
+Follow these [step-by-step store submission instructions](https://docs.microsoft.com/office/dev/store/add-in-submission-guide). When creating a new submission, specify you're submitting a Teams app.
+
+### App category mapping
```

Comment:
Is this app category mapping still needed and valid?

Reply:
Included same in preparation for submission